### ISSUE TITLE
Handler for /login. Positive authorization

### ISSUE THREAD

**Issue by Snorri1986** [Snorri1986/MyVehicleServiceApp#28]
1(ready to commit)If {code:0} -> show alert with text "Authorization was successful";
1.1(ready to commit) create class AuthController with annotation @controller and @RequestMapping("/main/workdesk").
1.2(ready to commit) create method getWorkDesk. 
2(create sub-tasks) Redirect to "add_new_vehicle.jsp" if subscriber has 0 vehicles in database;
3. Deploy and test on local Tomcat;
4. Update documentation:
5. Deploy and test on Heroku;
6. Commit new version.<br>
<hr>
20.01.2022 regular daily backup on web : 531351c6466b754e0a367a5627149e65be395f85<br>
01.02.2022 regular daily backup on web: 3f98a6547123ffe89d62fbe76185954cc84ddabf<br>
02.02.2022 regular daily commit on web: 330a30b9076d870d3bfacbaad1ea26962f39c6a6

**Comment by Snorri1986** [comment]
[1:] in progress

**Comment by Snorri1986** [comment]
[1]:
Question on stackoverflow:
https://stackoverflow.com/questions/70777761/how-to-analyze-response-in-json-from-spring-boot-service-in-spring-mvc

**Comment by Snorri1986** [comment]
[1:] Try solution: https://stackoverflow.com/questions/35150806/pass-form-data-from-spring-controller-to-an-external-endpoint

**Comment by Snorri1986** [comment]
[1:] wait while #32  will be finished.

**Comment by Snorri1986** [comment]
[1:]
Modify public String myAuthController(@RequestBody MultiValueMap<String, String[]> formData)
1. Add public class LoginResponseModel to the models package in mvs-web;
2. Add and init variable `authUrlPath`;
3. Create first version on auth success page - `workdesk.jsp` -> page should return only <h2>Welcome on workdesk </h2> and 
alert('Authorization was successful');
4.  Analyze answer from API in myAuthController.
0 -> open workdesk.jsp(temporary);
-1 -> log error

**Comment by Snorri1986** [comment]
Task suspended. Continue after #33

**Comment by Snorri1986** [comment]
Udemy course SpringMVC video -> 112

**Comment by Snorri1986** [comment]
Task in progress

**Comment by Snorri1986** [comment]
<b>Error:</b>
01-Feb-2022 23:09:04.912 WARNING [http-nio-8080-exec-3] org.springframework.web.servlet.DispatcherServlet.noHandlerFound No mapping for POST /mvs-web/workdesk

**Comment by Snorri1986** [comment]
[1]: 
1) create class AuthController with annotation @Controller and @RequestMapping("/main/workdesk").Status: in progress
2)  create method getWorkDesk. Status: suspended

**Comment by Snorri1986** [comment]
Task suspended. Continue after https://github.com/Snorri1986/MyVehicleServiceApp/issues/35

**Comment by Snorri1986** [comment]
#35 is done

**Comment by Snorri1986** [comment]
#34 is done

**Comment by Snorri1986** [comment]
closed. Will be new task
